certbot/main.py

```diff
@@ -394,13 +394,44 @@ def _init_le_client(config, authenticator, installer):
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
 
+def _deactivate(config, accounts, messenger):
+    if len(accounts) == 0:
```

Comment:
stylistic nit: `if not accounts:` should be used according to our [style guides](https://certbot.eff.org/docs/contributing.html#coding-style).